cmake/ChibiOS/18.2/ChibiOS_RTOS.cmake

```diff
@@ -0,0 +1,93 @@
+FOREACH (FAMILY F0 L0 L4)
+    SET (CHIBIOS_SOURCES_${CHIBIOS_KERNEL}_${FAMILY}
+        os/common/startup/ARMCMx/compilers/GCC/crt0_v6m.S
+        os/common/ports/ARMCMx/chcore.c
+        os/common/ports/ARMCMx/chcore_v6m.c
+        os/common/ports/ARMCMx/compilers/GCC/chcoreasm_v6m.S
+    )
+ENDFOREACH()
+
+FOREACH (FAMILY F1 F2 F3 F4 F7)
+    SET (CHIBIOS_SOURCES_${CHIBIOS_KERNEL}_${FAMILY}
+        os/common/startup/ARMCMx/compilers/GCC/crt0_v7m.S
+        os/common/ports/ARMCMx/chcore.c
+        os/common/ports/ARMCMx/chcore_v7m.c
+        os/common/ports/ARMCMx/compilers/GCC/chcoreasm_v7m.S
+    )
+ENDFOREACH()
+
+FOREACH (FAMILY F0 F1 F2 F3 F4 F7 L0 L1)
+    SET (CHIBIOS_INCLUDES_${CHIBIOS_KERNEL}_${FAMILY}
+        os/common/startup/ARMCMx/devices/STM32${FAMILY}xx
+        os/common/ext/ST/STM32${FAMILY}xx
+        os/common/oslib/include
+        os/common/ports/ARMCMx
+        os/common/ports/ARMCMx/compilers/GCC
+    )
+ENDFOREACH()
+
+SET (CHIBIOS_SOURCES_${CHIBIOS_KERNEL}
+    os/common/startup/ARMCMx/compilers/GCC/crt1.c
+    os/common/startup/ARMCMx/compilers/GCC/vectors.S
+)
+
+SET (CHIBIOS_INCLUDES_${CHIBIOS_KERNEL}
+    os/license
+    os/common/portability/GCC
+    os/common/startup/ARMCMx/compilers/GCC
+    os/common/ext/ARM/CMSIS/Core/Include
+)
+
+SET (CHIBIOS_SOURCES_${CHIBIOS_KERNEL}_MAILBOXES  os/common/oslib/src/chmboxes.c)
+SET (CHIBIOS_SOURCES_${CHIBIOS_KERNEL}_MEMCORE    os/common/oslib/src/chmemcore.c)
+SET (CHIBIOS_SOURCES_${CHIBIOS_KERNEL}_HEAP       os/common/oslib/src/chheap.c)
+SET (CHIBIOS_SOURCES_${CHIBIOS_KERNEL}_MEMPOOLS   os/common/oslib/src/chmempools.c)
+SET (CHIBIOS_SOURCES_${CHIBIOS_KERNEL}_FACTORY    os/common/oslib/src/chfactory.c)
```

Comment:
Not the easiest name 😅

Reply:
Well it comes from the template I used :(